packages/web-components/fast-components-msft/src/menu-item/menu-item.styles.ts

```diff
@@ -0,0 +1,134 @@
+import {
+    disabledCursor,
+    display,
+    focusVisible,
+    forcedColorsStylesheetBehavior,
+} from "@microsoft/fast-foundation";
+import {
+    heightNumber,
+    neutralFillStealthActiveBehavior,
+    neutralFillStealthHoverBehavior,
+    neutralFillStealthRestBehavior,
+    neutralFocusBehavior,
+    neutralForegroundRestBehavior,
+} from "../styles/index";
+import { SystemColors } from "@microsoft/fast-web-utilities";
+
+export const MenuItemStyles = css`
+    ${display("grid")} :host {
+        outline: none;
+        box-sizing: border-box;
+        height: calc(${heightNumber} * 1px);
+        grid-template-columns: 42px auto 42px;
+        grid-template-rows: auto;
+        justify-items: center;
+        align-items: center;
+        padding: 0;
+        margin: 0 calc(var(--design-unit) * 1px);
+        white-space: nowrap;
+        overflow: hidden;
+        color: ${neutralForegroundRestBehavior.var};
+        fill: ${neutralForegroundRestBehavior.var};
+        cursor: pointer;
+        font-size: var(--type-ramp-base-font-size);
+        line-height: var(--type-ramp-base-line-height);
+        border-radius: calc(var(--corner-radius) * 1px);
+        border: calc(var(--outline-width) * 1px) solid transparent;
+    }
+
+    :host(:${focusVisible}) {
+        border: calc(var(--outline-width) * 1px) solid ${neutralFocusBehavior.var};
+        box-shadow: 0 0 0 calc((var(--focus-outline-width) - var(--outline-width)) * 1px) ${
+            neutralFocusBehavior.var
+        };
+    }
+
+    :host(:hover) {
+        background: ${neutralFillStealthHoverBehavior.var};
+    }
+
+    :host(:active) {
+        background: ${neutralFillStealthActiveBehavior.var};
+    }
+
+    :host(.disabled) {
+        cursor: ${disabledCursor};
+        opacity: var(--disabled-opacity);
+    }
+
+    :host(.disabled:hover) .start,
+    :host(.disabled:hover) .end,
+    :host(.disabled:hover)::slotted(svg) {
+        fill: ${neutralForegroundRestBehavior.var};
+    }
+
+    .content {
+        grid-column-start: 2;
+        justify-self: start;
+        overflow: hidden;
+        text-overflow: ellipsis;
+    }
+
+    .start,
+    .end,
+    ::slotted(svg) {
+        ${
+            /* Glyph size and margin-left is temporary - 
+            replace when adaptive typography is figured out */ ""
+        } width: 16px;
+        height: 16px;
+    }
+
+    :host(:hover) .start,
+    :host(:hover) .end,
+    :host(:hover)::slotted(svg),
+    :host(:active) .start,
+    :host(:active) .end,
+    :host(:active)::slotted(svg) {
+        fill: ${neutralForegroundRestBehavior.var};
+    }
+`.withBehaviors(
+    neutralFillStealthActiveBehavior,
+    neutralFillStealthHoverBehavior,
+    neutralFillStealthRestBehavior,
+    neutralFocusBehavior,
+    neutralForegroundRestBehavior,
+    forcedColorsStylesheetBehavior(
+        css`
+            :host {
+                border-color: transparent;
+                forced-color-adjust: none;
```

Comment:
To also make sure the colors are correct at rest in high contrast, on `:host`, could you also add,
```
color: ${SystemColors.ButtonText};
fill: ${SystemColors.ButtonText};
```

Reply:
Thanks Kham!